app/src/main/res/values-fa/strings.xml

```diff
@@ -0,0 +1,106 @@
+    <string name="cardinal_direction_east_abbreviation">شرق</string>
+    <string name="cardinal_direction_south_abbreviation">جنوب</string>
+    <string name="cardinal_direction_west_abbreviation">غرب</string>
+
+    <string name="sensor_accuracy_no_contact">حسگر جهت‌یابی در دسترس نیست</string>
+    <string name="sensor_accuracy_unreliable">مقادیر حسگر جهت‌یابی دقیق نیستند</string>
+    <string name="sensor_accuracy_low">دقت پایین</string>
+    <string name="sensor_accuracy_medium">دقت متوسط</string>
+    <string name="sensor_accuracy_high">حداکثر دقت</string>
+    <string name="sensor_error_message">حسگر جهت‌یابی در دسترس نیست</string>
+    <string name="sensor_calibration_text">شما می‌توانید دقت جهت‌یابی را با چرخش دستگاه به شکل 8 بهبود دهید.</string>
+
+    <string name="location_loading">در انتظار به‌روزرسانی موقعیت</string>
+    <string name="location_error_message">موقعیت در دسترس نیست</string>
+    <string name="location_not_present">موقعیت در دسترس نیست. بازگشت به شمال مغناطیسی</string>
+    <string name="location_reload">بارگزاری مجدد موقعیت</string>
+    <string name="access_location_permission_rationale_title">برای دسترسی به موقعیت به مجوز نیاز داریم.</string>
+    <string name="access_location_permission_rationale_message">برای نشان دادن شمال واقعی، این برنامه نیاز به تعیین انحراف مغناطیسی بر اساس مکان فعلی شما دارد.</string>
+    <string name="access_location_permission_denied">اجازه دسترسی به موقعیت داده نشده است. بازگشت به سمت شمال مغناطیسی.</string>
+
+    <string name="settings">تنظیمات</string>
+
+    <string name="true_north">شمال واقعی</string>
+    <string name="true_north_summary">تصحیح انحراف مغناطیسی برای نمایش جهت شمال واقعی. این تنظیم نیاز به دسترسی به موقعیت مکانی شما دارد.</string>
+    <string name="magnetic_north">شمال مغناطیسی</string>
+
+    <string name="haptic_feedback">بازخورد لرزشی</string>
+    <string name="haptic_feedback_summary">با تغییر جهت قطب‌نما خواهد لرزید.</string>
+
+    <string name="display">نمایش</string>
+    <string name="night_mode">پوسته</string>
+    <string name="night_mode_follow_system">تعیین توسط سیستم</string>
+    <string name="night_mode_no">روشن</string>
+    <string name="night_mode_yes">تاریک</string>
+
+    <string name="about">درباره</string>
+    <string name="author">نویسنده</string>
+    <string name="license">مجوز</string>
+    <string name="source_code">سورس کد</string>
+    <string name="version">نسخه</string>
+
+    <string name="error_message">An error occurred.\nMessage: %1$s\nCode: %2$s</string>
+
+    <string name="error">خطا</string>
+    <string name="ok">باشه</string>
+    <string name="no_thanks">نه ممنون</string>
+
+    <string name="source_code_name" translatable="false">GitHub</string>
+    <string name="license_name" translatable="false">GNU GPLv3</string>
+    <string name="author_name" translatable="false">Philipp Bobek</string>
+
+    <string name="zero_degrees" translatable="false">0°</string>
```

Comment:
If you want you can remove all these strings with `translatable="false"` from your file.

Reply:
I will take care of that later on